src/board/ui.ts

```diff
@@ -63,12 +72,47 @@ export class BoardUI {
     return this.sensorsById.get(id);
   }
 
-  initialize(options: BoardOptions) {
-    this.audio.initialize(options);
+  initializeWebAssembly() {
+    this.operations.initialize();
+  }
+
+  initialize() {
+    this.audio.initialize({
+      defaultAudioCallback: this.operations.defaultAudioCallback!,
+      speechAudioCallback: this.operations.speechAudioCallback!,
+    });
     this.buttons.forEach((b) => b.initialize());
     this.pins.forEach((p) => p.initialize());
     this.display.initialize();
     this.accelerometer.initialize();
+    this.serialInputBuffer = [];
+  }
+
+  async stop(): Promise<void> {
+    await this.operations.stop();
+    window.board.serialInputBuffer.push(3, 4);
```

Comment:
This is madness. We are "board" and this is also happening inside operations.stop() where it is correctly sequenced.